sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroidOptions.java

```diff
@@ -33,6 +37,48 @@
   /** Enable or disable automatic breadcrumbs for App Components Using ComponentCallbacks */
   private boolean enableAppComponentBreadcrumbs = true;
 
+  /**
+   * Enables the Auto instrumentation for Activity lifecycle tracing.
+   *
+   * <ul>
+   *   <li>It also requires setting {@link SentryOptions#getTracesSampleRate()} or {@link
+   *       SentryOptions#getTracesSampler()}.
+   * </ul>
+   *
+   * <ul>
+   *   It starts a transaction before each Activity's onCreate method is called
+   *   (onActivityPreCreated).
+   *   <li>The transaction's name is the Activity's name, e.g. MainActivity.
+   *   <li>The transaction's operation is navigation.
+   * </ul>
+   *
+   * <ul>
+   *   It finishes the transaction after each activity's onResume method is called
+   *   (onActivityPostResumed), this depends on {@link
+   *   SentryAndroidOptions#enableAutoActivityLifecycleTracingFinish}.
+   *   <li>If {@link SentryAndroidOptions#enableAutoActivityLifecycleTracingFinish} is disabled, you
+   *       may finish the transaction manually.
+   *   <li>If the transaction is not finished either automatically or manually, we finish it
+   *       automatically after each Activity's onDestroy method is called (onActivityPostPaused).
+   *   <li>The transaction status will be {@link SpanStatus#OK} if none is set.
+   * </ul>
+   *
+   * The transaction is automatically bound to the {@link Scope}, but only if there's no transaction
+   * already bound to the Scope.
+   */
+  private boolean enableAutoActivityLifecycleTracing = true;
+
+  /**
+   * Enables the Auto instrumentation for Activity lifecycle tracing, but specifically when to
+   * finish the transaction, read {@link SentryAndroidOptions#enableAutoActivityLifecycleTracing}.
+   *
+   * <p>If you require a specific lifecycle to finish a transaction or even after the Activity is
+   * fully rendered but still waiting for an IO operation, you could call {@link ISpan#finish()}
+   * yourself on {@link Sentry#getSpan()}, be sure that you've finished all of your manually created
+   * Spans.
+   */
+  private boolean enableAutoActivityLifecycleTracingFinish = false;
```

Comment:
This means the user must call `finish` manually?
Shouldn't this be `true` by default?

Reply:
yes, it should, I was testing and commit/push wrongly, thx